xbmc/video/VideoDatabase.cpp

```diff
@@ -11900,61 +11900,62 @@ void CVideoDatabase::GetVideoVersions(VideoDbContentType itemType,
   }
 }
 
-void CVideoDatabase::GetDefaultVideoVersion(VideoDbContentType itemType, int dbId, CFileItem& item)
+int CVideoDatabase::GetDefaultVideoVersionId(VideoDbContentType itemType, int dbId)
 {
   if (!m_pDB || !m_pDS)
-    return;
+    return -1;
 
   MediaType mediaType;
-  std::string strSQL;
+  std::string query;
 
   if (itemType == VideoDbContentType::MOVIES)
```

Comment:
just a cosmetic, but could we invert the condition here? So instead of
```
if (itemType == VideoDbContentType::MOVIES) {
   // some logic
} else {
  return -1;
}
```
better use this logic
```
if (itemType != VideoDbContentType::MOVIES)
   return -1;

 // some logic
```

the current code is not deeply nested, but it's always better to avoid unnecessary code nesting if possible.
Just noticed that you could even move this condition also into the previous ` if (!m_pDB || !m_pDS)` condition and clean up the code even more.

Reply:
This is for future extension (TV Shows/Seasons extras, Episode versions/extras, etc.) for Video Versions feature. All implementation related with this future extension is implemented in this way.

Reply:
I'd say keep it clean for now - nobody knows how the data structure will look like in future. But that's just my take on it